### PR TITLE
[log] Add option to disable code gen

### PR DESCRIPTION
**Pull request by sumnulu**
ref: https://github.com/wvlet/airframe/issues/151

### CODE REVIEW

**Comment by xerial** [comment]
@sumnulu Thanks for the PR. Considering the characteristic of this compile-time only option, having less code coverage for this patch is OK.

File: airframe-log/README.md [hunk 1]
```diff
@@ -262,6 +262,13 @@ if(logger.isDebugEnabled) {
 Log message String generation will not happen unless the debug log is effective. 
 Scala macro is also used for finding source code location (LogSource).
 
+### To disable Logging at compile-time
+It is possible to disable the generation of logging statements at compile time.
+Add `scalacOptions += "-Xmacro-settings:wvlet.log.disable.[REPLACE_WITH_LEVEL]"` compiler option to `build.sbt`
```
Comment (xerial) [inline airframe-log/README.md]: The meaning of REPLACE_WITH_LEVEL is not clear without looking LocMacros code. Can we use a more clearer naming? For example, SUPPRESS_BELOW = {ERROR, WARN, INFO, DEBUG}: 

If SUPPRESS_BELOW = INFO, then DEBUG, TRACE log code generation will be suppressed.

Comment (sumnulu) [reply airframe-log/README.md]: Thanks for the feed back. Is it OK now?

Comment (xerial) [reply airframe-log/README.md]: Yes. LGTM now

**Pull request merged**